.github/workflows/statistician-image.yml

```diff
@@ -50,21 +57,23 @@ jobs:
       run: |
         echo "VERSION=$(cat statistician/version.txt)" >> $GITHUB_ENV
 
-    - name: Build and Push unstable Docker Image from PR / Push to Master
-      uses: whoan/docker-build-with-cache-action@v4
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
       with:
-        context: statistician
-        image_name: ${{ env.IMAGE_NAME }}
-        username: ${{ secrets.DOCKER_USER }}
-        password: ${{ secrets.DOCKER_PASS }}
-        image_tag: latest,${{ env.VERSION }}
+        role-to-assume: arn:aws:iam::538673716275:role/github-ecr
```

Comment:
Should this be hard-coded or passed in as an environment variable?

Reply:
Is it ever going to be different or reused elsewhere? If not, maybe hard-coding is okay?

Reply:
exposing aws account id is aweful

Reply:
all right...though I don't understand why...the whole point of having an assume role and using GitHub openid is to avoid storing the credentials as secrets. So I'll expect exposing account is not a big deal? I could be ignorant, and I'll take advice. Can I have somebody with privilege adding the thing as secret for me then?

Reply:
@omad

Reply:
There's been huge amounts of contradictory information in the past about exposing AWS Account IDs. They're sometimes hidden, but, they're exposed in all sorts of places, and are NOT considered sensitive.

As a case in point for this account, we make available public SNS topics for anyone to subscribe to new datsets, and the ARN for SNS Topics includes the account id.

Refs:
- [AWS SNS Notifications — Digital Earth Australia 1.0.0 documentation](https://docs.dea.ga.gov.au/setup/AWS/data_and_metadata.html#notifications)
- [Are AWS account IDs sensitive information? - Last Week in AWS Blog](https://www.lastweekinaws.com/blog/are-aws-account-ids-sensitive-information/)
- [Are AWS Account IDs Sensitive Information? - YouTube](https://www.youtube.com/watch?v=gYJgDymP_GU)